### PR TITLE
Build manifest for org.xfce.mousepad

### PR DESCRIPTION
**Pull request by Tamaranch**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
  - Using Portal would break some features of the application
  - Not allowing access to the entire file system for a text editor would be frustrating for the user
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
  - I'm not sure but I guess it's ok : Xfce is a well known open source project
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Tamaranch** [comment]
I did a forced push against the wrong branch in #2528, sorry.

File: org.xfce.mousepad.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  - /lib/girepository-1.0
+  - /lib/pkgconfig
+  - /include
+  - /share/gir-1.0
+  - /share/gtk-doc
+  - /share/man
+
+modules:
+  # build-time dependencies
+  - name: intltool
```
Comment (barthalion) [inline org.xfce.mousepad.yml:26]: Please switch to the version from shared-modules: https://github.com/flathub/shared-modules/blob/master/intltool/intltool-0.51.json

**Comment by barthalion** [comment]
> Not allowing access to the entire file system for a text editor would be frustrating for the user

Using the filesystem portal actually means any file can be passed because of user consent, but it's not a blocker.

bot, build org.xfce.mousepad

**Comment by barthalion** [comment]
Should we invite someone more from Xfce team to the Flathub repo?

**Comment by Tamaranch** [comment]
> Should we invite someone more from Xfce team to the Flathub repo?

Not for now, thanks.

**Comment by barthalion** [comment]
bot, build org.xfce.mousepad

**Pull request closed without merging**